.swiftlint.yml

```diff
@@ -12,6 +12,7 @@ opt_in_rules:
  - convenience_type
  - discouraged_optional_boolean
  - discouraged_optional_collection
+ - duplicate_imports
```

Comment:
FYI This and `unused_setter_value` are not opt-in rules.

```
$ swiftlint rules | grep -e opt-in -e duplicate_imports -e unused_setter_value
| identifier                               | opt-in | correctable | enabled in your config | kind        | analyzer | configuration |
| duplicate_imports                        | no     | no          | yes                    | idiomatic   | no       | warning       |
| unused_setter_value                      | no     | no          | yes                    | lint        | no       | warning       |
```

Reply:
Huh, didn't know that. Man they should note that (consistently) in the changelog.